build/MSBuildToolset/csc

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+THISDIR=$(dirname $0)
+
+chmod +x $THISDIR/corerun 2>/dev/null
+$THISDIR/corerun $THISDIR/csc.exe "$@"
```

Comment:
This line implies that csc.exe is going to be written to this directory.  That feels wrong as it's build deliberately modifying the source tree.